Assignment 1/index.html

```diff
@@ -0,0 +1,115 @@
+            </p>
+            <p style="text-align:center"><img src="images/russia.jpg" alt="russia flag"></p>
+        </article>
+
+        <aside>
+            <table>
+                <thead>
+                    <tr>
+                        <th>Country Name</th>
+                        <th>Population</th>
+                        <th>Capital</th>
+                        <th>Largest City</th>
+                        <th>moto</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td>India</td>
+                        <td>25cr</td>
+                        <td>New Delhi</td>
+                        <td>Mumbai</td>
+                        <td>Satyameva Jayate</td>
+                    </tr>
+                    <tr>
+                        <td>USA</td>
+                        <td>327,167,434</td>
+                        <td>Washington D.C.</td>
+                        <td>New York City</td>
+                        <td>In god we trust</td>
+                    </tr>
+                    <tr>
+                        <td>China</td>
+                        <td>1,403,500,365</td>
+                        <td>Beijing</td>
+                        <td>Shanghai</td>
+                        <td>March of the Volunteers</td>
+                    </tr>
+                    <tr>
+                        <td>Russia</td>
+                        <td>1,46,793,744</td>
+                        <td>Moscow</td>
+                        <td>Moscow</td>
+                        <td>Gosudarstvenny gimn Rossiyskoy Federatsii</td>
+                    </tr>
+                </tbody>
+                <tfoot></tfoot>
+            </table>
+
+            <div>
+                <form action="#" method="post">
+                    <label name="Name"><b>Name:</b></label>
```

Comment:
correct it, name should be in lower case

Reply:
Done.